.github/workflows/bench-turbopack.yml

```diff
@@ -0,0 +1,579 @@
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Rust
+        uses: ./.github/actions/setup-rust
+        with:
+          components: clippy
+          targets: wasm32-unknown-unknown
+
+      - name: Setup Go
+        uses: ./.github/actions/setup-go
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Run cargo check release
+        run: |
+          RUSTFLAGS="-D warnings -A deprecated" cargo groups check turbopack --features rustls-tls --release
+
+  turbopack_rust_test_bench1:
+    needs: [turbopack_rust_check]
+    runs-on: ubuntu-latest-16-core-oss
+    name: Turbopack Rust testing benchmarks on ubuntu
+
+    env:
+      TURBOPACK_BENCH_COUNTS: "100"
+      TURBOPACK_BENCH_PROGRESS: "1"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Rust
+        uses: ./.github/actions/setup-rust
+        with:
+          save-cache: true
+
+      - name: Setup Go
+        uses: ./.github/actions/setup-go
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Setup Node.js
+        uses: ./.github/actions/setup-node
+        with:
+          node-version: 18
+
+      - name: Build benchmarks for tests
+        timeout-minutes: 120
+        run: |
+          cargo tp-bench-test --no-run
+
+      - name: Run cargo test on benchmarks
+        timeout-minutes: 120
+        run: |
+          cargo tp-bench-test
+
+      - name: Build benchmarks for tests for other bundlers
+        if: needs.determine_jobs.outputs.turbopack_bench == 'true'
+        timeout-minutes: 120
+        run: |
+          cargo test --benches --release -p turbopack-bench --no-run
+
+      - name: Run cargo test on benchmarks for other bundlers
+        if: needs.determine_jobs.outputs.turbopack_bench == 'true'
+        timeout-minutes: 120
+        run: |
+          cargo test --benches --release -p turbopack-bench
+
+  turbopack_rust_test_bench2:
+    needs: [turbopack_rust_check]
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - name: macos
+            runner: macos-latest
+          # Temporarily disable windows bench due to consistent timeouts
+          # - name: windows
+          #   runner: windows-2019
+    runs-on: ${{ matrix.os.runner }}
+    name: Turbopack Rust testing benchmarks on ${{ matrix.os.name }}
+
+    env:
+      TURBOPACK_BENCH_COUNTS: "100"
+      TURBOPACK_BENCH_PROGRESS: "1"
+
+    steps:
+      - name: Set git to use LF
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
+        if: matrix.os.name == 'windows'
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Rust
+        uses: ./.github/actions/setup-rust
+        with:
+          save-cache: true
+
+      - name: Setup Go
+        uses: ./.github/actions/setup-go
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Not sure why Go needs to be setup for this job, but not planning to change it in this PR since it's a copy paste of before. I don't want to break something that is working.